.github/workflows/linter.yml

```
@@ -0,0 +1,15 @@
+name: Linter
+on:
+  pull_request:
+    branches: [ master ]
+jobs:
+  linter:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+
```

```suggestion

    - uses: actions/setup-node@v1
      with:
        node-version: '10'

```

Otherwise, we don't know what version we get and we want to ensure that it works at least with 10 (or maybe 12).